src/Microsoft.ML.Auto/API/AutoCatalog.cs

```diff
@@ -0,0 +1,242 @@
+        }
+
+        /// <summary>
+        /// Creates a new AutoML experiment to run on a multiclass classification dataset.
+        /// </summary>
+        /// <param name="maxExperimentTimeInSeconds">Maximum number of seconds that experiment will run.</param>
+        /// <returns>A new AutoML multiclass classification experiment.</returns>
+        /// <remarks>
+        /// An experiment may run for longer than <paramref name="maxExperimentTimeInSeconds"/>.
+        /// This is because once AutoML starts training an ML.NET model, AutoML lets the
+        /// model train to completion. For instance, if the first model 
+        /// AutoML trains takes 4 hours, and the second model trained takes 5 hours,
+        /// but <paramref name="maxExperimentTimeInSeconds"/> was the number of seconds in 6 hours, 
+        /// the experiment will run for 4 + 5 = 9 hours (not 6 hours).
+        /// </remarks>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[MulticlassClassificationExperiment](~/../docs/samples/docs/samples/Microsoft.ML.Samples/AutoML/MulticlassClassificationExperiment.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public MulticlassClassificationExperiment CreateMulticlassClassificationExperiment(uint maxExperimentTimeInSeconds)
+        {
+            return new MulticlassClassificationExperiment(_context, new MulticlassExperimentSettings()
+            {
+                MaxExperimentTimeInSeconds = maxExperimentTimeInSeconds
+            });
+        }
+
+        /// <summary>
+        /// Creates a new AutoML experiment to run on a binary classification dataset.
+        /// </summary>
+        /// <param name="experimentSettings">Settings for the AutoML experiment.</param>
+        /// <returns>A new AutoML multiclass classification experiment.</returns>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[MulticlassClassificationExperimentWithExperimentSettings](~/../docs/samples/docs/samples/Microsoft.ML.Samples/AutoML/MulticlassClassificationExperimentWithExperimentSettings.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public MulticlassClassificationExperiment CreateMulticlassClassificationExperiment(MulticlassExperimentSettings experimentSettings)
+        {
+            return new MulticlassClassificationExperiment(_context, experimentSettings);
+        }
+
+        /// <summary>
+        /// Infers information about the columns of a dataset in a file located at <paramref name="path"/>.
+        /// </summary>
+        /// <param name="path">Path to a dataset file.</param>
+        /// <param name="labelColumnName">The name of the label column.</param>
+        /// <param name="separatorChar">The character used as separator between data points in a row. If <see langword="null"/>, AutoML will try to infer this value.</param>
```

Comment:
I think of an entire row as (potentially) a singular multi-dimensional "data point", perhaps "data element" is better?
```suggestion
        /// <param name="separatorChar">The character used as separator between data elements in a row. If <see langword="null"/>, AutoML will try to infer this value.</param>
```

Reply:
this is much better -- thank you!